names/names.py

```diff
@@ -10,11 +10,27 @@
 names_2 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
-duplicates = []
+names = {}
+
```

Comment:
Good job keeping this simple Kyle, while enhancing the existing code. We talked about the use a `set`, and your solution does that in a way by building up the `names` `dict` with values and checking against that as you loop over `names_2` values.